lib/GenomeFileUtil/core/GenomeToGenbank.py

```diff
@@ -144,8 +151,19 @@ def __init__(self, genome=None):
         self._features = self._ga.get_features()
 
         print('writing file')
-
-        contigs = self._ga.get_assembly().get_contigs()
+        # read in fasta file to build the contig index
```

Comment:
Seems like this block should be something the assemblyutils or assemblyapi could provide.  Probably okay for now.  Just thinking about the case where we have to re-scan an entire plant genome to get this info when we already have it in the assembly object.  Right?